src/covariances.jl

```diff
@@ -0,0 +1,80 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+using Statistics
```

Comment:
All imports should go in the main source file.

Reply:
Done in commit 27ba036744f365d651d56aa0e76c1fffc4114c40, but review is needed.